openc3/bin/openc3cli

```diff
@@ -648,6 +650,59 @@ def run_bridge(filename, params)
   end
 end
 
+def await_dotter(script_id)
+  OpenC3::RunningScriptWebSocketApi.new(id: script_id) do |api|
+    while (resp = api.read) do
+      case resp['type']
+      when 'file'
+        puts resp.pretty_inspect
+      when 'line'
+        fn = resp['filename'].nil? ? '<no file>' : resp['filename']
+        puts "At [#{fn}:#{resp['line_no']}] state [#{resp['state']}]"
+      when 'output'
+        puts resp['line']
+      when 'time'
+        puts resp.pretty_inspect
+      when 'complete'
+        puts 'script complete'
+        break
+      else
+        puts resp
+      end
+      sleep(2)
+    end
+  end
+end
+
+def await_script(args=[])
+  #  we are limiting the wait for status, not the script run time
+  #  we want 0 to mean 'forever'
+  #  REVIEW:       do we want to accept suite, scope, and environment options at the cli?
```

Comment:
Maybe play with running a suite file and see what makes sense. We do want to support scope and disconnect and passing specific ENV vars.

Reply:
Okay, I'll find a suite to try now that the single-script code is a little cleaner.  I'll want to chat about scope, env and disconnect briefly.

Reply:
It just runs the suite until its components are started, then exits.